src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,63 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.25.0.dev2 (01/17/2020)
+------------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Add Bandit Python linter (#8929)
+  `PR #8929 <https://github.com/pantsbuild/pants/pull/8929>`_
+
+* Tweak options parser to allow globs in filesystem specs (#8981)
```

Comment:
This is more a refactoring. Filesystem specs will still fail in this dev release. This is prework for adding that feature.